examples/example.krun

```diff
@@ -75,5 +75,5 @@ TEMP_READ_PAUSE = 1
 #PRE_EXECUTION_CMDS = ["sudo service cron stop"]
 #POST_EXECUTION_CMDS = ["sudo service cron start"]
 
-# CPU pinning (off by default)
-#ENABLE_PINNING = False
+# CPU pinning (on by default)
+#ENABLE_PINNING = True
```

Comment:
Point taken about a man page. Until the thing gets too unwieldy (which will be a while), I'd prefer a single doc page though.